index.html

```diff
@@ -346,5 +346,51 @@ <h2 class="about-header">About<br />Myself</h2>
         </li>
       </ul>
     </section>
+    <footer class="footer">
+      <div class="layer"></div>
+      <div class="contact-me-first">
+        <h2 class="contact-me-header">Contact me</h2>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <div class="form-container">
+        <form action="https://formspree.io/f/#" method="post">
+          <ul class="form-list">
+            <li>
```

Comment:
[Optional]

- All the `ul` and `li` is unnecessary here since you have already used form to wrap all the input field, there is no need of extra grouping.

Reply:
I removed all unnecessary ul and li tags from my form.